terraform/projects/app-ecs-albs/main.tf

```diff
@@ -169,17 +169,15 @@ resource "aws_route53_record" "alerts_alias" {
   type    = "A"
 
   alias {
-    name                   = "${aws_lb.nginx_auth_external_alb.dns_name}"
-    zone_id                = "${aws_lb.nginx_auth_external_alb.zone_id}"
+    name                   = "${aws_lb.monitoring_external_alb.dns_name}"
+    zone_id                = "${aws_lb.monitoring_external_alb.zone_id}"
     evaluate_target_health = false
   }
 }
 
-resource "aws_lb_target_group" "nginx_auth_external_endpoint" {
-  count = "${local.infra_network_public_subnets_count}"
-
-  name                 = "${var.stack_name}-ext-tg-${count.index + 1}"
-  port                 = 80
+resource "aws_lb_target_group" "nginx_auth_proxy_external_endpoint" {
+  name                 = "${var.stack_name}-ext-tg"
+  port                 = 8181
```

Comment:
why port 8181? why not a more standard port like 80 or 8080?  You don't open the port directly on the host (using `hostPort` so I think 80 and 8080 are available to you.  Going with 8181 violates the principle of least surprise to me - it jumps out as being unusual, and it makes me wonder if there's something funny going on.

Reply:
This was not to clash with the existing endpoint I think, it can be changed to 80 now that we have removed the old listeners and ALB